### DISCUSSION TITLE
404 For "express gratitude"

### DISCUSSION THREAD

**Discussion by holmanb** [pradyunsg/furo#599] [Potential Issue] [answered]
FYI Looks like the link for "[express gratitude](https://shoutouts.dev/projects/pradyunsg/furo)" is broken.

I'll do it this way:

Thanks!

:-)

**Comment by pradyunsg** [comment, marked as answer]
https://github.com/pradyunsg/furo/commit/205e1b3d91475e0a2851fee94e9609fd4ee84c09 for removing the broken link.
